server/database.py

```diff
@@ -47,10 +48,12 @@ def status(self, value):
         self.status_level = User.STATUS_LIST.index(value)
 
     @classmethod
-    def get(cls, id, *args, **kwargs):
+    def get(cls, id, name="Unknown", *args, **kwargs):
```

Comment:
As I pointed out in the other PR, should possibly set `user.name = Character.get(id).name`.

Reply:
as I replied in the other, do we actually want to call Character.get in the constructor> We can, but I like constructors fast and simple when possible